docs/source/filters.rst

```diff
@@ -3,7 +3,14 @@ Filters
 
 The filter language provides a set of classes that can be used to construct complex
 queries for filtering data. Each filter class represents a specific filtering criterion,
-allowing users to tailor their queries to their specific needs. Below is an overview of the
+allowing users to tailor their queries to their specific needs. 
+
+The filter can be used on any container property. These can be references directly as 
+``('space', 'view_external_id/view_version', 'property')``,
+but it is usually more convenient to use the ``as_property_ref`` method on the View or ViewID objects as:
+``myView.as_property_ref('property')``.
```

Comment:
This jumps straight into Data Modeling specifics - we'll need some more context first. We should start by stating that the filter classes are shared between Data Modeling and classic resources, then we can continue to the different ways of composing the property references.